en/installation/web-and-post-installation.md

```diff
@@ -134,5 +134,5 @@ Go to `Administration > Extensions > Manager` menu and click on
 
 ## Getting started
 
-Go to the [Getting Started](../getting-started/installation-first-steps.html#start-to-monitor-your-first-host)
+Go to the [Getting Started](../getting-started/installation-first-steps.html#request-your-free-trial)
```

Comment:
should the getting started link redirect to request your trial ?

Reply:
May be it could be better than starting by requesting a free trial. But starting by adding a linux serveur in the monitoring without login and a token request, it's completely dummy. 

Step by step !